os/board/rtl8721csm/src/component/common/mbed/targets/hal/rtl8721d/wait_api.c

```diff
@@ -18,12 +18,12 @@
 
 //extern void DelayUs(u32 us);
 extern void DelayUs(unsigned int us);
-
+#ifndef CONFIG_SCHED_HAVE_PARENT

```

Comment:
Why do you enclose this function with config? If needed, `wait_ms()` is okay not to enclose?

Reply:
Because it is clashing with our TizenRT wait() API in <kernel/sched/sched_wait.c> due to which the tc_sched_wait testcase is failing.
As wait_ms() is not a prototype in TizenRT, I feel we dont need to exclude it. What is your opinion ??

Reply:
1. It looks it is the same as sleep. Could you confirm this function is not used in rtk codes for that purpose?
2. The conditionals should be as below.
```c
#if !(defined(CONFIG_SCHED_WAITPID) && defined(CONFIG_SCHED_HAVE_PARENT))
```
3. Let's add the location where you modify in title.
```
board/rtl8721csm: xxxx
```

Reply:
As discussed with realtek, have removed the file wait_api.c
Modified the commit title as well!